build infra failed on task.fendermint-deps

### Issue type

Bug

### Have you reproduced the bug with the latest dev version?

Yes

### Version

dev

### Custom code

Yes

### OS platform and distribution

MacOS

### Describe the issue

Failed building ipc infrastructure on task.fendermint-deps. After a few tests and debugging,  I think the issue is around `CARGO_MAKE_WORKSPACE_WORKING_DIRECTORY` [here](https://github.com/consensus-shipyard/fendermint/blob/05cd5e77721a44bd5d0665edd2052762937fc0be/infra/scripts/fendermint.toml#L67). 
Since `scripts/fendermint.toml` is copied from fendermint project into `ipc/bin/ipc-infra`, the  `CARGO_MAKE_WORKSPACE_WORKING_DIRECTORY` param was expected as the directory for fendermint codebase so that `make docker-build` can be run to build images for fendermint if it does not exist in your machine. But after moving fendermint into `ipc/bin/ipc-infra`, the `CARGO_MAKE_WORKSPACE_WORKING_DIRECTORY` changes to `<local folder>/ipc` where fendermint makefile cannot be found to run `make docker-build`. 

### Repro steps

- make sure to delete the fendermint docker image from your machine. 
- run `cargo make --makefile bin/ipc-infra/Makefile.toml fendermint-deps` 

btw this task works fine in fendermint project, but not in ipc project.

I add few lines in `ipc/bin/ipc-infra/scripts/fendermint.toml` to test this task.
```
[tasks.fendermint-deps]
script = """
# Check if the image exists
# TODO: Check the version or use a flag to always re-build?
if docker images | awk '{print $1":"$2}' | grep fendermint; then
    echo fendermint image already exists
    docker images | grep fendermint
else
    echo fendermint image does not exists
    echo dir=${CARGO_MAKE_WORKSPACE_WORKING_DIRECTORY}
    cd ${CARGO_MAKE_WORKSPACE_WORKING_DIRECTORY}
    make docker-build
fi
"""
```

### Relevant log output

```shell
[cargo-make] INFO - Running Task: fendermint-deps
fendermint image does not exists
dir=/Users/longfeiwang/Desktop/IPC/ipc
make: *** No rule to make target `docker-build'.  Stop.
```

--- a/docs/quickstart-calibration.md
+++ b/docs/quickstart-calibration.md
@@ -168,7 +168,7 @@ Remember the address of your bootstrap for the next step. This address has the f
 ./bin/ipc-cli subnet add-bootstrap --subnet=<SUBNET_ID> --endpoint=<BOOTSRAP_ENDPOINT>
 ```
 
-* The bootstrap nodes currently deployed in the network can be queried through the following command: 
+* The bootstrap nodes currently deployed in the network can be queried through the following command:
 ```bash
 ./bin/ipc-cli subnet list-bootstraps --subnet=<SUBNET_ID>
 ```
@@ -178,9 +178,9 @@ With the bootstrap node deployed and advertised to the network, we are now ready
 
 * First we need to export the private keys of our validators from the addresses that we created with our `ipc-cli wallet` to a known path so they can be picked by Fendermint to sign blocks. We can use the default repo of IPC for this, `~/.ipc`.
 ```bash
-./bin/ipc-cli wallet export -w evm -a <WALLET_ADDR1> --fendermint -o ~/.ipc/<PRIV_KEY_VALIDATOR_1>
-./bin/ipc-cli wallet export -w evm -a <WALLET_ADDR1> --fendermint -o ~/.ipc/<PRIV_KEY_VALIDATOR_1>
-./bin/ipc-cli wallet export -w evm -a <WALLET_ADDR1> --fendermint -o ~/.ipc/<PRIV_KEY_VALIDATOR_1>
+./bin/ipc-cli wallet export -w evm -a <WALLET_ADDR1> --hex -o ~/.ipc/<PRIV_KEY_VALIDATOR_1>
+./bin/ipc-cli wallet export -w evm -a <WALLET_ADDR1> --hex -o ~/.ipc/<PRIV_KEY_VALIDATOR_1>
+./bin/ipc-cli wallet export -w evm -a <WALLET_ADDR1> --hex -o ~/.ipc/<PRIV_KEY_VALIDATOR_1>
 ```
 
 * Now we have all that we need to deploy the three validators using the following command (configured for each of the validators, i.e. replace the arguments with `<..-n>` to fit that of the specific validator).
@@ -188,7 +188,7 @@ With the bootstrap node deployed and advertised to the network, we are now ready
 ```bash
 cargo make --makefile /bin/ipc-infra/Makefile.toml \
     -e NODE_NAME=validator-<n> \
-    -e VALIDATOR_PRIV_KEY=<PATH_PRIV_KEY_VALIDATOR_n> \
+    -e PRIVATE_KEY_PATH=<PATH_PRIV_KEY_VALIDATOR_n> \
     -e SUBNET_ID=<SUBNET_ID> \
     -e CMT_P2P_HOST_PORT=<COMETBFT_P2P_PORT_n> -e CMT_RPC_HOST_PORT=<COMETBFT_RPC_PORT_n> \
     -e ETHAPI_HOST_PORT=<ETH_RPC_PORT_n> \

--- a/docs/usage.md
+++ b/docs/usage.md
@@ -20,12 +20,19 @@ The `ipc-cli` has internally an EVM wallet that it uses to sign transactions and
 ```
 ```console
 # Sample execution
-./bin/ipc-cli wallet export -w evm -a 0x406a7a1d002b71ece175cc7e067620ae5b58e9ec -o /tmp/priv.key                                                                                     ✔  12:21:38 
-exported new wallet with address 0x406a7a1d002b71ece175cc7e067620ae5b58e9ec in file "/tmp/priv.key"```
+./bin/ipc-cli wallet export -w evm -a 0x406a7a1d002b71ece175cc7e067620ae5b58e9ec -o /tmp/priv.key
+exported new wallet with address 0x406a7a1d002b71ece175cc7e067620ae5b58e9ec in file "/tmp/priv.key"
+```
 
-* You can also export your private key in a format that can be consumed by Fendermint by adding the `--fendermint` flag.
+* You can also export your private key encoded in base64 in a format that can be consumed by Fendermint by adding the `--fendermint` flag.
 ```bash
 ./bin/ipc-cli wallet export -w evm -a <EVM-ADDRESS> -o <OUTPUT_FILE> --fendermint
+```
+
+* Or hex encoded as expected by Ethereum tooling (like Metamask or hardhat).
+```bash
+./bin/ipc-cli wallet export -w evm -a <EVM-ADDRESS> -o <OUTPUT_FILE> --hex
+```
 
 * Importing a key from a file
 ```bash

--- a/ipc/cli/src/commands/wallet/export.rs
+++ b/ipc/cli/src/commands/wallet/export.rs
@@ -27,6 +27,10 @@ impl WalletExport {
             .get(&address.into())?
             .ok_or_else(|| anyhow!("key does not exists"))?;
 
+        if arguments.hex {
+            return Ok(hex::encode(key_info.private_key()));
+        }
+
         if arguments.fendermint {
             return Ok(BASE64_STANDARD.encode(key_info.private_key()));
         }
@@ -43,9 +47,15 @@ impl WalletExport {
 
         let addr = Address::from_str(&arguments.address)?;
         let key_info = wallet.write().unwrap().export(&addr)?;
+
+        if arguments.hex {
+            return Ok(hex::encode(key_info.private_key()));
+        }
+
         if arguments.fendermint {
             return Ok(BASE64_STANDARD.encode(key_info.private_key()));
         }
+
         Ok(serde_json::to_string(&LotusJsonKeyType {
             r#type: WalletKeyType::try_from(*key_info.key_type())?.to_string(),
             private_key: BASE64_STANDARD.encode(key_info.private_key()),
@@ -87,7 +97,7 @@ impl CommandLineHandler for WalletExport {
 }
 
 #[derive(Debug, Args)]
-#[command(about = "Export the key from a wallet address")]
+#[command(about = "Export the key from a wallet address in JSON format")]
 pub(crate) struct WalletExportArgs {
     #[arg(long, short, help = "Address of the key to export")]
     pub address: String,
@@ -102,9 +112,11 @@ pub(crate) struct WalletExportArgs {
     #[arg(
         long,
         short,
-        help = "Only returns the secret key in base64 as Fendermint expects"
+        help = "Exports the secret key encoded in base64 as Fendermint expects"
     )]
     pub fendermint: bool,
+    #[arg(long, short, help = "Export the hex encoded secret key")]
+    pub hex: bool,
 }
 
 pub(crate) struct WalletPublicKey;

--- a/scripts/install_infra.sh
+++ b/scripts/install_infra.sh
@@ -19,15 +19,10 @@ fi
 
 build_infra() {
     echo "[*] Building fendermint..."
-    make build docker-build
     cd $PWD
 
     echo "[*] Updating infra scripts..."
     cp -r $infra_path/fendermint/infra/* $infra_path
-    # TODO: This will no longer be necessary once https://github.com/consensus-shipyard/fendermint/pull/329
-    # is merged
-    mkdir -p ./target/release
-    mv $infra_path/fendermint/target/release/fendermint $PWD/target/release
 }
 
 # Function to display help message

